helm_deploy/values-dev.yaml

```diff
@@ -13,6 +13,9 @@ generic-service:
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
+  allowlist:
+    dxw-vpn: "54.76.254.148/32"
```

Comment:
these is an override from the `values.yaml` so should have everything in there otherwise you'll just be allowing `dxw` access to dev - probably not what you meant...